client/my-sites/patterns/components/pattern-preview/style.scss

```diff
@@ -47,8 +47,8 @@
 	}
 
 	.pattern-preview__header {
-		align-items: start;
 		display: flex;
+		align-items: flex-start;
```

Comment:
Looking at [this data](https://developer.mozilla.org/en-US/docs/Web/CSS/align-items#browser_compatibility), I don't think browser compatibility is a concern with using `align-items: start`

Reply:
Hm, agree 👌
Let's go with `start` and for consistency let's try to use it everywhere in the future :)